### DISCUSSION TITLE
Enabling jvm and js linters in qodana.yml

### DISCUSSION THREAD

**Discussion by sjivan** [JetBrains/Qodana#197] [Q&A] [answered]
Hi,
I'd like to enable Java and JS code inspection.

If I add this line in qodana.yml with linter: jetbrains/qodana-jvm:latest the CI/CD pipeline as call as CLI runs the Java inspections but the ESlint inspection that I have enabled is skipped.

If I set linter to linter: jetbrains/qodana-js:2023.2 the CI/CD pipeline and CLI runs the ESlint inspection but the Java inspections are skipped.

How do I enable both?

Additionally when I do a local Qodana run from IntelliJ IDEA the ESlint inspection is skipped. Is this expected? Would I need to run from Webstorm if I want local runs to inspect JS via ESlint?

Thanks!

**Comment by Cazayus** [comment, marked as answer]
I'm pretty sure you can't have both in a single run. 
You're supposed to run qodana-jvm for java and qodana-js for JS/TS.

It means that you need to have multiple qodana.yml file. 
You can do this by having a qodana_jvm.yml and qodana-js.yml. 
You then mount one or the other when running docker (with something like -v <workspace_loc>/qodana_jvm.yaml:/data/project/qodana.yaml)
